content/sigs/advocacy-and-outreach/outreach-programs/index.adoc

```diff
@@ -10,9 +10,11 @@ tags:
   - community
   - outreach
   - programs
+opengraph:
+  image:/images/gsoc/jenkins-gsoc-logo_small.png
```

Comment:
I cannot see the image in the test page, I do not know why. How is the opengraph suppose to operate?

Reply:
Got it. Thanks!